test/functional/uuid/jsonb.ts

```diff
@@ -7,10 +7,17 @@ import {createTestingConnections, closeTestingConnections} from "../../utils/tes
 describe("uuid type", () => {
 
     let connections: Connection[];
-    before(async () => connections = await createTestingConnections({
-        entities: [Record],
-        enabledDrivers: ["postgres"] // because only postgres supports jsonb type
-    }));
+    before(async () => {
+        connections = await createTestingConnections({
+            entities: [Record],
+            enabledDrivers: ["postgres"] // because only postgres supports jsonb type
+        });
+
+        connections.forEach(async connection => {
+            let queryRunner = await connection.driver.createQueryRunner();
+            await queryRunner.query(`CREATE extension IF NOT EXISTS "uuid-ossp"`);
```

Comment:
In order to use the generate_uuid functions, you have to enable the uuid-ossp extension.  You may be able to configure travis to do this, but, even still, anyone who runs the tests locally will need to enable them manually.  It may be worth having the schema synchronizer do this automatically, but I haven't even looked at how that works yet.